upi/openstack/post-install.yaml

```diff
@@ -0,0 +1,14 @@
+- hosts: all
+  gather_facts: no
+
+  tasks:
+  - name: 'Create snat policy file'
+    template:
+      src: cluster_snat_policy.conf.j2
+      dest: cluster_snatpolicy.yaml
+
+  - name: 'Create cluster SNAT policy'
+    k8s:
+      state: present
+      kubeconfig: "{{ kubeconfig }}"
```

Comment:
do we have {{kubeconfig}} declared in inventory?

Reply:
yes, user will have to add it under aci_cni['kubeconfig']